### PR TITLE
Add support for the lpc11xx_32

### PR DESCRIPTION
**Pull request by c1728p9**
Add the flash algo and target information for the lpc11xx_32.

### CODE REVIEW

**Comment by 0xc0170** [comment]
Thanks, we will test this hopefully soon enough

File: pyOCD/target/target_lpc11xx_32.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from cortex_m import CortexM
+from .memory_map import (FlashRegion, RamRegion, MemoryMap)
```
Comment (0xc0170) [inline pyOCD/target/target_lpc11xx_32.py]: Looks like all our target files have the same as these 2 lines, one is absolute one is relative.. We might want to clean this in pyOCD

**Comment by 0xc0170** [comment]
+1

File: pyOCD/target/target_lpc11xx_32.py (hunk 1 above)
Comment (c1728p9) [inline pyOCD/target/target_lpc11xx_32.py]: sounds good, I'll take care of this in a separate pull request.

Comment (0xc0170) [inline pyOCD/target/target_lpc11xx_32.py]: +1

**Pull request merged**